queries/git_config/highlights.scm

```diff
@@ -0,0 +1,46 @@
+; Sections
+
+(section_name) @type
+
+((section_name) @function.builtin
+ (#eq? @function.builtin "include"))
+
+((section_header
+   (section_name) @function.builtin
+   (subsection_name))
+ (#eq? @function.builtin "includeIf"))
```

Comment:
```suggestion
((section_name) @include
 (#eq? @include "include"))

((section_header
   (section_name) @include
   (subsection_name))
 (#eq? @include "includeIf"))
```

Reply:
Good catch..I mostly copied this from Helix and didn't pay attention there :laughing: